### ISSUE TITLE
Pending reviews are tied to indivudual so can not be shared among teachers before being submitted

### ISSUE THREAD

**Issue by jelmstrom** [github/gh-classroom#11]
When grading assignments the comments made by me as a teacher in a pending review are hidden from any other teachers that are involved in the course, meaning that it is hard to share workloads and collaborate on correction. 
it would be great if reviews are public to teachers, and can be submitted by anyone

**Comment by jelmstrom** [comment]
Hi
I think its even before that.

it appears that model that the classroom works on now is that its _one_
teacher per class. We had several. What happens then is that each
instructor starts their own review, which means you can not look at the
suggestions other instructors have made.
/* Johan */



Den tis 26 sep. 2023 kl 15:24 skrev Shane K. Panter <
***@***.***>:

**Comment by jelmstrom** [comment]
Yes, thats what we discussed as well, but that approach was not so easy
either, as we'd have to switch logged in accounts frequently to work on
personal contributions to other repositories.
If a pending review would be accessible to a team or a group of people, it
would make life easier for teachers that are part of a group.

Regards

Johan.



Den tis 26 sep. 2023 kl 16:50 skrev Shane K. Panter <
***@***.***>:

**Comment by shanep** [comment]
Hi @jelmstrom, you are not alone regarding this feature :)  You can read about the [discussion](https://github.com/orgs/community/discussions/10369) regarding how reviews in the pending state can't be seen by anyone else or shared until they are submitted. 

Unfortunately this feature is not a part of classrooms.github.com but of github itself. 

-Shane
